bip122.md

```diff
@@ -0,0 +1,134 @@
+---
+namespace-identifier: bip122
+title: BIP122 Namespace
+author: Simon Warta (@webmaster128), ligi <ligi@ligi.de>, Pedro Gomes (@pedrouid)
+discussions-to: https://github.com/ChainAgnostic/namespaces/pulls/3
+status: Draft
+type: Standard
+created: 2022-02-09
+updated: 2022-02-09
+requires: CAIP-2, CAIP-10
+supercedes: CAIP-4
```

Comment:
```suggestion
supersedes: CAIP-4
```

Reply:
i "resolved" it because i realized it should be "replaces" instead of "supersedes" anyways--- although I should probably add `superseded-by: namespaces-bip122` in CAIP-4, unless we just delete the namespace-CAIPs!